### PR TITLE
Wk 194 detail panel header style

### PR DESCRIPTION
**Pull request by aditudorache**
This PR fixes the styling of the header in the detail panel after adding the navigation link

### CODE REVIEW

File: src/components/detailPanel/DetailPanel.js [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
+
 import { spotTypeDisplayNames } from '../../constants';
+
 import { trackDownload } from 'helpers';
 import UserContext from '../../shared/user/UserContext';
 
+import { Heading, Button, Link } from '@datapunt/asc-ui';
+
+import { Close } from '@datapunt/asc-assets';
+import { HeaderStyle, ContentStyle } from './DetailPanelStyle';
```
Comment (jasperswart) [inline src/components/detailPanel/DetailPanel.js]: ik denk dat het beter is als je deze niet  in een andere file zet

File: src/components/detailPanel/DetailPanel.js [hunk 2]
```diff
@@ -50,23 +58,32 @@ const DetailPanel = ({ isOpen, togglePanel, feature }) => {
       documents,
     } = feature.properties;
     const [lng, lat] = feature.geometry.coordinates;
+
     return (
       <div
         className={classNames(
           styles.Container,
           isOpen ? styles.ContainerOpen : ''
         )}
```
Comment (jasperswart) [inline src/components/detailPanel/DetailPanel.js]: ik vind die hele classNames werkwijze heel onhandig. wat vind jij?

Comment (aditudorache) [reply src/components/detailPanel/DetailPanel.js]: not my choice, normally I have no problem with it, but I try to use only styled components

File: src/components/detailPanel/DetailPanel.js (hunk 1 above)
Comment (aditudorache) [reply src/components/detailPanel/DetailPanel.js]: One of the reasons is that I want to skip the <ComponentName>Style.js from testing/coverrage. The other reason is that the number of style classes can be large and the overview of the component becomes difficult to read

**Pull request merged**